docker-compose.yml

```diff
@@ -7,5 +7,4 @@ services:
       - "4000:4000"
     volumes:
       - .:/usr/src/app
-    working_dir: /usr/src/app
```

Comment:
I wasnt able to get your branch to work. After playing with it more, it seems to, in part, be this line. When I add it back in, make preview seems to work fine and the website renders fine at :4000. When this line is removed, I get an immediate:

`make: *** No rule to make target 'preview'.  Stop.`

As I dont use Docker often, its entirely possible that Im doing something wrong.